cmd/create/machinepool/options_test.go

```diff
@@ -0,0 +1,59 @@
+package machinepool
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	mpOpts "github.com/openshift/rosa/pkg/options/machinepool"
+	"github.com/openshift/rosa/pkg/reporter"
+)
+
+var _ = Describe("CreateMachinepoolOptions", func() {
+	var (
+		machinepoolOptions *CreateMachinepoolOptions
+		userOptions        *mpOpts.CreateMachinepoolUserOptions
+	)
+
+	BeforeEach(func() {
+		machinepoolOptions = NewCreateMachinepoolOptions()
+		userOptions = NewCreateMachinepoolUserOptions()
+	})
+
+	Describe("NewCreateMachinepoolUserOptions", func() {
```

Comment:
@den-rgb Minor, but I think we _tend_ to use `Context` under the initial `Describe`. No biggie, and please disregard if there is precedent else where.